dotnet/examples/OZWForm/src/ConfigurationWakeUpDlg.cs

```diff
@@ -1,14 +1,39 @@
+//-----------------------------------------------------------------------------

```

Comment:
Where did you copy the header from? 
The example apps should be under GPL license, and only the libary code under LGPL